### PR TITLE
feature: prettify xml junit report

### LINKED ISSUE #165: Prettify the Generated Junit Report (RDT-324)

### PR DESCRIPTION
**Pull request by alekseiapa**
- prettify xml junit report
close #165

### CODE REVIEW

**Comment by alekseiapa** [comment]
@hfudev Could you please take a look ?

**Comment by hfudev** [comment]
LGTM. could you set it to a CLI option? and make it non default. the default one should still be the minified version.

btw, please also `close #165`

**Comment by alekseiapa** [comment]
@alekseiapa Could you please take a look ? I'll close #165 once this PR got merged

**Comment by hfudev** [comment]
You may add this inside the PR description to auto-close the related issue.

File: pytest-embedded/pytest_embedded/utils.py [hunk 1]
```diff
@@ -80,6 +81,13 @@ def find_by_suffix(suffix: str, path: str) -> List[str]:
     return res
 
 
+def prettify_xml(file_path: str):
```
Comment (hfudev) [inline pytest-embedded/pytest_embedded/utils.py:84]: this function will never be reused... how about we move it to the `plugin.py` instead?

**Review by hfudev — CHANGES_REQUESTED** [review]
last comment. Thank you!

File: pytest-embedded/pytest_embedded/utils.py (hunk 1 above)
Comment (alekseiapa) [reply pytest-embedded/pytest_embedded/utils.py]: Sure, I've moved the function to the correct place.

**Pull request merged**